go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da
 	github.com/aws/aws-sdk-go v1.29.8
 	github.com/bmizerany/pat v0.0.0-20160217103242-c068ca2f0aac
+	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e
```

Comment:
I think we need to do a quick review of this library.

Reply:
It seems to do what it says on the tin when I looked at it - picks chars off stdin and buffers then into a line of text, with smart processing to handle navigation